src/index.html

```diff
@@ -14,23 +14,20 @@ <h1>Leader Board</h1>
             <div class="score-list">
                 <div class="score-list-header">
                  <h2>Recent Scores</h2>
-                <button>Refresh</button>
+                <button type="button" id="refresh">Refresh</button>
                 </div>
              <div class="list-of-scores">
-                <ul>
-                    <li>Nati 220</li>
-                    <li>Nati 220</li>
-                    <li>Nati 220</li>
-                    <li>Nati 220</li>
-                </ul>
+                <div id="scores">
+                    
+                </div>
              </div>   
             </div>
             <div class="add-score">
                 <h2>Add your Score</h2>
                 <input type="text" name="" id="finput">
-                <input type="text" name="" id="">
+                <input type="number" name="" id="linput">
```

Comment:
Please make sure to add placeholder texts for your inputs to improve UI/UX.

[OPTIONAL] Consider adding validations as well to prevent users from submitting empty scores.